### DISCUSSION TITLE
Install in project with react version 18.0.0

### DISCUSSION THREAD

**Discussion by YuriyAlekseev9567** [TanStack/table#3827] [General]
How to install a package in an application that uses React version 18.0.0
![Снимок экрана 2022-04-13 224738](https://user-images.githubusercontent.com/84084334/163258525-527f2e95-e037-4229-a656-ac91e0f6e230.png)

**Comment by WaelNalouti** [comment]
Try this:
`npm install --legacy-peer-deps react-table`
